src/LibraryManager/Providers/Cdnjs/CdnjsCatalog.cs

```diff
@@ -214,7 +214,7 @@ private IEnumerable<CdnjsLibraryGroup> GetSortedSearchResult(string term)
 
         private static string NormalizedGroupName(string groupName)
         {
-            string cleanName = AliasedName(groupName);
+            string cleanName = groupName;
```

Comment:
No need for this assignment. You can directly use groupName below.

Reply:
Will do